pkg/agent/apiserver/handlers/networkpolicy/handler.go

```diff
@@ -41,7 +41,7 @@ func HandleFunc(aq agentquerier.AgentQuerier) http.HandlerFunc {
 		var nps []cpv1beta.NetworkPolicy
 
 		if npFilter.Pod != "" {
-			interfaces := aq.GetInterfaceStore().GetContainerInterfacesByPod(npFilter.Pod, npFilter.Namespace)
+			interfaces := aq.GetInterfaceStore().GetInterfacesByEntity(npFilter.Pod, npFilter.Namespace)
```

Comment:
I saw you still changed apiserver and Traceflow to use GetInterfacesByEntity(). Could we limit the change to only NetworkPolicy controller?

Reply:
Yes I am still working on the PR. Will let you know once I push a commit for this

Reply:
Done. Agent controller should be the only one using `GetInterfacesByEntity` now